lib/gollum/views/page.rb

```diff
@@ -184,6 +192,42 @@ def content_without_page_header(content)
         # .inner_html will cause href escaping on UTF-8
         doc.css("div#gollum-root").children.to_xml(@@to_xml)
       end
+
+      def table(data)
+        result = "<table>\n"
+        case data
+          when Hash
```

Comment:
Not a big fan of explicit class checking in Ruby. How desirable is it to allow `Array`s to work?